packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -79,25 +191,42 @@ const Bulletin = ({
   ctaText,
   ctaLink,
   liveText,
-}) => (
-  <BulletinWrapper type={type}>
-    {image && type === 'video' && image}
-    <BulletinHeading script={script} service={service} type={type}>
-      <VisuallyHiddenText>
-        {isLive ? `${ctaText} ${liveText} ` : `${ctaText} `}
-      </VisuallyHiddenText>
-      {isLive && <LiveLabel service={service}>{liveText}</LiveLabel>}
-      <Link href={ctaLink}>{headlineText}</Link>
-    </BulletinHeading>
-    <BulletinSummary script={script} service={service} type={type}>
-      {summaryText}
-    </BulletinSummary>
-    <PlayCTA isLive={isLive} service={service} script={script}>
-      <IconWrapper>{mediaIcons[type.toLowerCase()]}</IconWrapper>
-      {ctaText}
-    </PlayCTA>
-  </BulletinWrapper>
-);
+  dir,
+}) => {
+  const isAudio = type === 'audio';
+  const BulletinWrapper = isAudio ? RadioBulletinWrapper : TVBulletinWrapper;
+  const ImageWrapper = isAudio ? RadioImageWrapper : TVImageWrapper;
+  const TextWrapper = isAudio ? RadioTextWrapper : TVTextWrapper;
+  const BulletinHeading = isAudio ? RadioHeading : TVHeading;
+
+  return (
+    <BulletinWrapper>
+      <ImageWrapper>{image}</ImageWrapper>
+      <TextWrapper direction={dir}>
```

Comment:
Using `dir` as a prop in `TextWrapper` causes the `Bulletin` to shift to the right in `rtl`

<img width="1679" alt="Screenshot 2019-11-21 at 10 00 36" src="https://user-images.githubusercontent.com/48688870/69316637-51efba80-0c4a-11ea-8846-08cc2acbd0aa.png">